model/schema/core.yaml

```diff
@@ -733,12 +735,14 @@ slots:
       Object representing the organization (e.g. MOD) from which the data was
       sourced and a CrossReference to that organisation's site
     multivalued: false
+    required: true
     range: DataProvider
 
   data_provider_dto:
     description: >-
       Ingest object representing the organization (e.g. MOD) from which the data
       was sourced and a CrossReference to that organisation's site
+    required: true
```

Comment:
Does this have to be required? Are we going to set it or are the DQM's going to set it? I think it would be better for us to get it?

Reply:
For disease annotations we require DQMs to set it - I would prefer to keep it like this for everything.  For bulk submission via data loads we can determine the data provider from the bulk load type, but for direct API submissions we can't.  Also, submission by DQMs allows the cross-reference part of data provider to be submitted.